leveldb.c

```diff
@@ -507,6 +509,9 @@ static inline leveldb_options_t* php_leveldb_get_open_options(zval *options_zv,
 
 		if (comparator) {
 			*out_comparator = comparator;
+		} else {
+			zval_ptr_dtor(z2);
+			efree(z2);
```

Comment:
This is probably redundant, I'm not aware of any failure conditions for creating a comparator